dali/operators/image/remap/warp.h

```diff
@@ -229,10 +229,10 @@ class Warp : public SequenceOperator<Backend> {
   const OpSpec &Spec() const { return this->spec_; }
 
  private:
-  /// @defgroup WarpStaticType Dynamic to static type routing
+  /** @defgroup WarpStaticType Dynamic to static type routing */
```

Comment:
Shouldn't this be a single `/**` and `*/` for the group of lines?

Reply:
Done. Actually, it's in class, so it should be named group